__tests__/__integration__/imperative.integration.test.ts

```diff
@@ -9,11 +9,11 @@
 *
 */
 
-import { runCliScript } from "../__src__/TestUtils";
+import { ITestEnvironment, runCliScript } from "../__packages__/cli-test-utils";
```

Comment:
Apologies if I already asked this, but would there be a problem if we decide to import from the `@zowe/cli-test-utils` package?

Reply:
At this level, I think so. Not sure why we have test files at this level though, that aren't part of any package?

Reply:
maybe we should move this test file somewhere inside a package 😋